spec/fields/access_condition_spec.rb

```diff
@@ -68,31 +60,19 @@ def mods_display_non_ignore_access_condition(mods_record)
         expect(fields.first.values.length).to eq(1)
         expect(fields.first.values.first).to include("<a href='http://creativecommons.org/licenses/by-sa/3.0/'>")
         expect(fields.first.values.first).to include(
-          'This work is licensed under a Creative Commons Attribution-Noncommercial 3.0 Unported License'
+          'This work is licensed under a Creative Commons Attribution-Share Alike 3.0 Unported License'
         )
       end
       it 'should itentify and link OpenDataCommons licenses properly' do
         fields = mods_display_access_condition(@odc_license_note).fields
         expect(fields.length).to eq(1)
         expect(fields.first.values.length).to eq(1)
-        expect(fields.first.values.first).to include("<a href='http://opendatacommons.org/licenses/pddl'>")
+        expect(fields.first.values.first).to include("<a href='http://opendatacommons.org/licenses/pddl/'>")
         expect(fields.first.values.first).to include(
           'This work is licensed under a Open Data Commons Public Domain Dedication and License (PDDL)'
         )
       end
-      it 'should have a configurable version for CC licenses' do
```

Comment:
Just linking this here so if/when we look at addressing this in PURL, we remember to do it here too:

https://github.com/sul-dlss/purl/issues/321